browser/main/NoteList/index.js

```diff
@@ -684,8 +684,11 @@ class NoteList extends React.Component {
       })
   }
 
-  copyNoteLink (note) {
-    const noteLink = `[${note.title}](:note:${note.key})`
+  copyNoteLink () {
+    const { selectedNoteKeys } = this.state
```

Comment:
I think you shouldn't re-find the selected note again because the selected note is already been searched at the top of the `handleNoteContextMenu` function. Please re-use the selected note in the `handleNoteContextMenu` function to improve performance for the app.